engine/src/main/java/org/terasology/rendering/nui/layouts/FlowLayout.java

```diff
@@ -121,6 +138,17 @@ public int getVerticalSpacing() {
      * @param spacing The spacing, in pixels
      * @return This {@code FlowLayout}
      */
+    public FlowLayout setHorizontalSpacing(int spacing) {
+        this.horizontalSpacing = spacing;
+        return this;
+    }
+
+    /**
+     * Sets the horizontal spacing between adjacent widgets in this {@code FlowLayout}.
```

Comment:
I think the docstrings are now flipped for `setHorizontalSpacing` and `setVerticalSpacing` 😅

Reply:
And I did double check... On it 👍